css-canvas-medical/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <div class="container">
+    <div class="row justify-content-spacebetween">
+      <div class="logo">
+        <img class="logo-img-width" src="images/logo-blue.svg" alt="logo-blue">
+      </div>
+      <nav>
+        <ul>
+          <li>
+            <a href="#">Product</a>
+          </li>
+          <li>
+            <a href="#">Implementation</a>
+          </li>
+          <li>
+            <a href="#">About Us</a>
+          </li>
+          <li>
+            <a href="#">Blog</a>
+          </li>
+          <li><button id="navbtn">Contact Us</button></li>
+        </ul>
+      </nav>
+    </div>
+  </div>
+  <hr />
+  <div class="container">
+    <div class="backgroundimg">
+      <div class="col-six-over-ten add-padding-80">
+        <h1 class="adjust-font-color-weight">The EMR Reimagined for Primary Care</h1>
+        <p class="adjust-font-color-size-family">Can't remember the last time you talked with a patient without focusing
+            on the EMR? Canvas lets you rediscover joy in the work you do. Capture meaningful data,
+            identify gaps in care, and move seamlessly between tasks within the platform. You stay present with
+            patients while your whole care team stays up-to-speed. Be a better doctor with Canvas, designed
+            for primary care providers with the help of primary care providers.
+        </p>
+        <button class="btn">Get Started</button>
+      </div>
+    </div>
+  </div>
+  <div class="locations">
+    <div class="row justify-content-spacebetween">
+      <div class="align-content-center col-two-fifth">
+        <h2 class="adjust-heading-font-color-size-family-weight">Clinicians Prefer Canvas</h2>
+        <p class="adjust-subheading-color-line-height">Physician-owned primary care practices across the country have already made Canvas the fabric of their care.</p>
+      </div>
+    </div>
+    <div class="row container justify-content-spacebetween">
```

Comment:
No one element should be both a `container` and a `row`. Instead, you should make an element with the `container` class that has a child with the `row` class

Reply:
This issue is present in multiple other places in your code, please make sure it is addressed in all instances.